.github/workflows/on-demand-vrfv2plus-smoke-tests.yml

```diff
@@ -70,6 +70,8 @@ jobs:
     with:
       custom_test_list_json: ${{ needs.set-tests-to-run.outputs.test_list }}
       chainlink_version: ${{ inputs.chainlink_version }}
+      slack_notification_after_tests: always
```

Comment:
and I would also add `slack_notification_after_tests_notify_user_id_on_failure` if it is set

Reply:
@iljapavlovs done, please check latest commits